v2g-liberty-cards/src/schedule-settings-card.ts

```diff
@@ -88,4 +89,27 @@ export class ScheduleSettingsCard extends LitElement {
       </div>
     `;
   }
+
+  // TODO: Fix in global.py and fm_client.py that hass.states[entityIds.fmConnectionStatus]
+  // get written with state.
+  private _renderFMConnectionStatus() {
+    const state = this._fmConnectionStatus.state;
+    const isConnected = state === 'Successfully connected';
+    const hasConnectionError =
+      state === 'Connection error' || state === 'Failed to connect';
+    const error = tp('connection-error');
+    const success = tp('connection-success', {
+      time: elapsedTimeSince(this._fmConnectionStatus.last_updated),
+    });
+    return isConnected
+      ? html`
+          <div class="success">
```

Comment:
Use `ha-alert` instead?

Reply:
No, in this case it renders not lake an alert but the box with a green line. This is not a standard HA element so keeping this HTML / style.